### PR TITLE
Add Solver Dashboard Config

### PR DESCRIPTION
**Pull request by bh2smith**
Based on pending PR to merge Dashboard Loading (https://github.com/bh2smith/duneapi/pull/22)

Dashboard config generated by the following command:

```sh
python -m dashboards.load_dashboard --dashboard-slug=GPv2-Solver-Info
```

copy the contents of the `out/` directory into `dashboards/`

Then you can also run

```sh
python -m dashboards.save_dashboard --dashboard-slug=GPv2-Solver-Info
```

to upsert the queries (but there is no need they are up to date). Note all of this needs to be done from the dashboard owner account.

I have renamed a `config.json` to `_config.json` so it appears at the top 
and also needed to change the query file path from `out/` to `dashboard/` (these two items will be fixed in the dependency soon).

### CODE REVIEW

File: dashboards/GPv2-Solver-Info/_config.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    },
+    {
+      "id": 571231,
+      "name": "Solver Yields",
+      "query_file": "./dashboards/GPv2-Solver-Info/solver-yields.sql",
+      "network": "Ethereum Mainnet",
+      "parameters": []
+    }
+  ]
+}
```
Comment (bh2smith) [inline dashboards/GPv2-Solver-Info/_config.json:86]: I will also have to update https://github.com/bh2smith/duneapi/ to add new lines at end of generated files.

File: dashboards/GPv2-Solver-Info/_config.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      "id": 38617,
+      "name": "Batch Settlement Info",
+      "query_file": "./dashboards/GPv2-Solver-Info/batch-settlement-info.sql",
+      "network": "Ethereum Mainnet",
+      "parameters": []
+    },
+    {
+      "id": 571231,
+      "name": "Solver Yields",
+      "query_file": "./dashboards/GPv2-Solver-Info/solver-yields.sql",
```
Comment (bh2smith) [inline dashboards/GPv2-Solver-Info/_config.json:81]: Will update the dependency repo to put this path `./dashboards/GPv2-Solver-Info` in the metadaata at the top of the config.

**Comment by josojo** [comment]
Looks good so far.

Maybe you can add a small readme into the repo, such that it is easier to use.
also, maybe you wanna proceed this PR with another PR that puts the existing js code into a subfolder. If we would merge it as is, it's strange to have this mixture of two unrelated projects.


I was trying to run your package with my dune dashboard:
https://dune.xyz/josojo/Gnosis-Auction
```

  File "/Users/alexherrmann/cow/cow-dune/env/lib/python3.10/site-packages/duneapi/dashboard.py", line 124, in from_dune
    return cls(
  File "/Users/alexherrmann/cow/cow-dune/env/lib/python3.10/site-packages/duneapi/dashboard.py", line 40, in __init__
    raise DuplicateQueryError(dupes)
duneapi.dashboard.DuplicateQueryError: ['SELECT COUNT(Distinct "userId")\r\nFROM gnosis_auction."EasyAuction_evt_NewSellOrder"', 'With \r\nall_auctions as (\r\nSELECT * FROM gnosis_auction."EasyAuction_evt_NewAuction")\r\nSELECT COUNT(*) FROM all_auctions']
```
I am not sure what it means. 

I only skimmed over the queries and they seem fine

**Comment by bh2smith** [comment]
This error means that your dashboard has the same query appearing twice (duplicate query) and so you could technically remove a tile from you dashboard in favour of a different visualization. When I get back to a computer I can find which one. 

Great idea about the readme! I also thought about separating the other project into a sub folder but decided against it at first. I can do this now that you have suggested it.

**Comment by bh2smith** [comment]
At first glance, my guess would be that you have the same query for two different networks. I hadn't considered that but could fix up the base dependency to be able to handle that.

File: dashboards/GPv2-Solver-Info/_config.json (hunk 1 above)
Comment (bh2smith) [reply dashboards/GPv2-Solver-Info/_config.json:86]: https://github.com/bh2smith/duneapi/issues/25

File: dashboards/GPv2-Solver-Info/_config.json (hunk 2 above)
Comment (bh2smith) [reply dashboards/GPv2-Solver-Info/_config.json:81]: https://github.com/bh2smith/duneapi/issues/26

**Comment by bh2smith** [comment]
@josojo - here is the PR fixing the issues described here: Once merged I can republish the new package version and update this PR.

https://github.com/bh2smith/duneapi/pull/30

File: README.md [hunk 3]
```diff
@@ -1,12 +1,83 @@
...
+## Dashboard Manager (Python)
+
+From within the dashboard directory:
+
+Setup python environment
+
+```sh
+python3 -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
```
[the remark this answers is not in the document]
Comment (bh2smith) [reply README.md:40]: I wrote in the sentence before "from within the dashboard directory" but I like the idea to put the readme in each project.

[the remark this answers is not in the document]
Comment (josojo) [reply README.md:40]: Ah sorry, I missed that. Yeah, maybe two readmes is the best solution, but not a must

File: dashboards/GPv2-Solver-Info/solver-failure-rates-by-environment.sql [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        sum(
+            CASE
+                WHEN success is false THEN 1
+                ELSE 0
+            END
+        ) as failures
+    FROM ethereum."transactions"
+    WHERE block_time > NOW() - INTERVAL '7 DAY'
+        AND (
+            "to" = '\x9008D19f58AAbD9eD0D60971565AA8510560ab41'
```
[the remark this answers is not in the document]
Comment (bh2smith) [reply dashboards/GPv2-Solver-Info/solver-failure-rates-by-environment.sql:19]: I think we can create a tiny table in dune abstractions with this data something like 

Settlement Contracts: [contract_address: bytea | version: str] with values

```
\x3328f5f2cEcAF00a2443082B657CedEAf70bfAEf, alpha
\x9008D19f58AAbD9eD0D60971565AA8510560ab41, beta
```

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bh2smith) [reply README.md]: geFixed

**Comment by bh2smith** [comment]
Partially fulfills epic issue #3

**Pull request merged**